exopy_qm/tasks/tasks/ResumeAndGetDataTask.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import numpy as np
+from inspect import cleandoc
+import time
+import qm.qua
+from atom.api import Float, Int, List, Typed, Str, Value, Bool, set_default
+from exopy.tasks.api import InstrumentTask
+
+logger = logging.getLogger(__name__)
+
+
+class ResumeAndGetDataTask(InstrumentTask):
+    """Resume a QM program and finishes it using the 'iterate' input stream.
+
+    """
+
+
+    database_entries = set_default({'Results': {}})
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+
+    def check(self, *args, **kwargs):
+        test, traceback = super(ResumeAndGetDataTask,
+                                self).check(*args, **kwargs)
+
+        return test, traceback
+
+    def perform(self):
+        # We assume that the program is paused for the last time
+        self.driver.resume()
+
+        self.driver.wait_for_all_results()
+        results = self.driver.get_results()
+        report = self.driver.get_execution_report()
```

Comment:
Didn't we have issues in the past requesting execution reports every time? I think I remember commenting out that line with Rémy but I'm not sure.

Reply:
Since this task is not supposed to be executed that frequently, I would say that it is ok ? I can comment it anyway

Reply:
Yes, if it's only once it's ok